packages/registry/profiles/generic/generic-fixed-hand.json

```diff
@@ -0,0 +1,32 @@
+{
+    "profileId" : "generic-fixed-hand",
+    "fallbackProfileIds": ["generic-hand", "generic-hand-select"],
```

Comment:
should we fall back to `generic-trigger` instead?

Reply:
Hm... I could see an argument for it either way. Curious what others think, especially @AdaRoseCannon.

Reply:
generic-trigger IIUC falls back to being basically a controller?

Reply:
yes. it would be strange to fall back to an animated hand that won't draw correctly

Reply:
What do you think about keeping `generic-hand-select` in the profiles array, since it's an accurate description of the input and doesn't have an associated asset?

Reply:
that sounds good to me

Reply:
> What do you think about keeping `generic-hand-select` in the profiles array, since it's an accurate description of the input and doesn't have an associated asset?

Yeah that makes sense, to me.

Reply:
Thanks for the feedback! Updated the profile list to `['generic-fixed-hand', 'generic-hand-select', 'generic-trigger']`, which was my sense of the best "Most specific->Least Specific" progression.